docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+- Users are content items as well, so your knowledge about content management is reused.
+
+See [User management product guide](user_management_guide.md) for more.
+
+#### Content access
+
+- The REST API and GraphQL API support access to and edition of the content.
+- [[= product_name_headless =]] offers a complete PHP API to extend the ways to access content.
+- A design engine and a view controller offer to create plain text content views (such as HTML, JSON, XML, CSS, JS, CSV, or Markdown),
+  and to factorize those views by using theme cascades.
+  This design engine is used in the Back Office which is equally extendable.
+- Multichannel, content can be accessed through several channel configurations,
+  such as the domain name it replies to, the sub-part of the content tree it starts from, the users rights, or the design theme.
+  The Back Office itself is such a channel.
+- Multi-repository, the same platform can use separate databases if impermeability is needed between channel groups.
+
+### Advanced features
+
+On top of this strong core, [[= product_name =]] Headless edition brings tools to increase user experience, from final front users to Back Office contributors.
+
+#### [[= product_name_headless =]] is a complete Digital Experience Platform (DXP)
+
+- [[= product_name_headless =]] comes with a personalization engine, allowing to recommend content to the front user according to its behavior,
+  or, when authenticated, contents matching user's segment/group. See [Personalization product guide](personalization_brochure.md) for more.
+- [[= product_name_headless =]]' scheduler allows to establish the future of the content and having a living front application with events even when the editorial team is absent or reduced. At midnight, during weekends or vacations, front users can discover new contents. A calendar summaries those scheduled content event. The calendar is extendable as everything in the Back Office, some event source could be added to coordinate content events with other company events.
+
+#### [[= product_name_headless =]] adds more ways to structure and organize your content
+
+- [[= product_name_headless =]] has specific feature to organize complex products and their catalogs, making it a strong Product Information Management (PIM) software.
+    - Product are organized per product types × variants × catalogs × categories × tags.
+    - Product attributes are grouped and factorized among product types. For example, fabric + color + size can be shared by many clothing product types.
+    - Product variants can rapidly be created by the automatic declination of attributes having a defined set of values.
+    - See [PIM product guide](pim_guide.md) for more.
+- [[= product_name_headless =]]' taxonomy feature allows to tag content items to organize them by topics in a much intuitive way for the editor than a content tree with multiple locations would.
```

Comment:
```suggestion
With [Taxonomy](https://doc.ibexa.co/projects/userguide/en/master/content_management/taxonomy/taxonomy/) you can tag content items to organize them by topics in a much intuitive way for the editor than a content tree with multiple locations would.
```

Reply:
I avoided to introduce too much link. I could link each feature to its user or dev doc page but half of the text will become blue, hard to read with too many rebounds. So, I choose to only link to other product guides.

```suggestion
- With taxonomy, you can tag content items to organize them by topics in a much intuitive way for the editor than a content tree with multiple locations would.
```